archlinuxcn/sendanywhere/lilac.yaml

```
@@ -0,0 +1,13 @@
+maintainers:
+  - github: Xeonacid
+    email: h.dwwwwww@gmail.com
+
+build_prefix: extra-x86_64
+
+pre_build: aur_pre_build
+
+post_build: aur_post_build
```

@lilydjwg also this?
```suggestion
pre_build: vcs_update

post_build: git_pkgbuild_commit
```